ocs_ci/deployment/deployment.py

```diff
@@ -174,23 +196,296 @@ def deploy_cluster(self, log_cli_level="DEBUG"):
+                s3_secret_data["metadata"]["name"] = secret_name
+                s3_secret_yaml = tempfile.NamedTemporaryFile(
+                    mode="w+",
+                    prefix=f"s3_secret_{s3_acm_secret_suffix}_{region}_",
+                    delete=False,
+                )
+                templating.dump_data_to_temp_yaml(s3_secret_data, s3_secret_yaml.name)
+                secret_yaml_files.append(s3_secret_yaml.name)
+
+            # Create s3 secret on all clusters
+            for secret_yaml in secret_yaml_files:
+                cmd = f"oc create -f {secret_yaml}"
+                run_cmd_multicluster(cmd)
+
+            # Create ODF HUB operator only on ACM HUB
+            dr_hub_operator_data = templating.load_yaml(
+                constants.OPENSHIFT_DR_HUB_OPERATOR
+            )
+            dr_hub_operator_yaml = tempfile.NamedTemporaryFile(
+                mode="w+", prefix="dr_hub_operator_", delete=False
+            )
+            templating.dump_to_temp_yaml(
+                dr_hub_operator_data, dr_hub_operator_yaml.name
+            )
+            run_cmd(f"oc create -f {dr_hub_operator_yaml.name}")
+
+            # Create DR policy on ACM hub cluster
+            dr_policy_hub_data = templating.load_yaml(constants.DR_POLICY_ACM_HUB)
+            # Update DR cluster name
+            for (cluster, name_entry) in zip(
+                get_non_acm_cluster_config(), dr_policy_hub_data["spec"]["drClusterSet"]
+            ):
+                name_entry["name"] = cluster.ENV_DATA["cluster_name"]
+            dr_policy_hub_yaml = tempfile.NamedTemporaryFile(
+                mode="w+", prefix="dr_policy_hub_", delete=False
+            )
+            templating.dump_to_temp_yaml(dr_policy_hub_data, dr_policy_hub_yaml.name)
+            run_cmd(f"oc create -f {dr_policy_hub_yaml.name}")
+
+            # Create ODF cluster operator on all non-acm clusters
+            dr_cluster_operator_data = templating.load_yaml(
+                constants.OPENSHIFT_DR_CLUSTER_OPERATOR
+            )
+            for cluster in get_non_acm_cluster_config():
+                config.switch_ctx(config.clusters.index(cluster))
+                dr_cluster_operator_yaml = tempfile.NamedTemporaryFile(
+                    mode="w+", prefix="dr_cluster_operator_", delete=False
+                )
+                templating.dump_to_temp_yaml(
+                    dr_cluster_operator_data, dr_cluster_operator_yaml.name
+                )
+                run_cmd(f"oc create -f {dr_cluster_operator_yaml.name}")
+            # Reset ctx
+            config.switch_acm_ctx()
+
+    def get_participating_regions(self):
+        """
+        Get all the participating regions in the DR scenario
+
+        Returns:
+            list of str: List of participating regions
+
+        """
+        # For first cut just returning east and west
+        return ["east", "west"]
```

Comment:
This we need to take out of configuration.

Reply:
I will update it in the next PR as MCG will be the default metadata object store.